src/Parser.svelte

```diff
@@ -41,14 +42,24 @@
           {/each}
         </svelte:component>
       </svelte:component>
-    {:else if type === 'list'}
+    {:else if type === 'list' && ordered}
+      {#if ordered}
       <svelte:component this={renderers.list} {...$$restProps}>
```

Comment:
Thanks a lot for your work!

If we do this, `ordered` is no longer being passed to the `list` renderer. It should be passed explicitly here.

Reply:
Added this to both branches